Tasks/HelmDeployV0/src/helmcommands/helminstall.ts

```diff
@@ -29,6 +29,7 @@ export function addArguments(helmCli: helmcli): void {
     var valueFile = tl.getInput("valueFile", false);
     var rootFolder = tl.getVariable('System.DefaultWorkingDirectory');
     var enableTls = tl.getBoolInput("enableTls", false);
+    var version = tl.getInput('version', false);
```

Comment:
Don't you want to add some version string validations? 
Like verifying if it follows the semVer format?

Reply:
done